src/main/resources/assets/computercraft/lua/rom/programs/size

```diff
@@ -0,0 +1,27 @@
+local Arg = {...}
+
+if Arg[1] == nil then
+  print("Usage: size <path>")
+  return 1
+end
+
+local filename = shell.resolve(Arg[1])
+
+if fs.exists(filename) == true then
+  if fs.isDir(filename) then
+    os.loadAPI("/usr/apis/wilmaapi")
+    local fileta = wilmaapi.listAllFiles(filename)
```

Comment:
You are loading some wierd api that you did not include here.
Also why not use fs.list(string path) and not fs.isDir() to identify files instead of this Api..

On sidenote if size needs to calculate directory size then it probably should use recursion on all directories that are in scanned directory.